src/DynamoCoreWpf/ViewModels/PackageManager/InstalledPackagesViewModel.cs

```diff
@@ -1,51 +1,261 @@
 ﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
 using System.Linq;
-
+using System.Security.Cryptography;
```

Comment:
whats this using about?

Reply:
I don't know how it got there. I will remove it.